Sage/carbon
# chore(menu-list): remove classic theme

Removal of classic theme from MenuList component. Redundant documentation has been deleted and
replaced with an MDX example.

BREAKING CHANGE: Classic theme has been removed from the MenuList component. This component no
longer supports the classic theme.

fixes FE-3114

### Proposed behaviour

Remove classic theme
- Delete redundant documentation
- Refactor story to be CSF
- Create an MDX example

### Current behaviour
- Component currently supports classic theme
- Documentation is inline with our previous standards
- Story is in the deprecated storiesOf format

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-3114-menulist_remove_classic_theme`
- `npm start` to start Storybook
- Navigate to MenuList story in Storybook
- Component should not have any classic theme
- Example of component should be in MDX format
